src/plugincollection.js

```diff
@@ -189,7 +190,7 @@ export default class PluginCollection {
 			const errorMsg = 'plugincollection-plugin-not-found: Some plugins are not available and could not be loaded.';
 
 			// Log the error so it's more visible on the console. Hopefully, for better DX.
-			log.error( errorMsg, { plugins: missingPlugins } );
+			console.error( attachLinkToDocumentation( errorMsg ), { plugins: missingPlugins } );
```

Comment:
This should be `CKEditorError` exception.

Reply:
We have an error below.

Reply:
:+1: